data/overlayfiles/motd-manager-server-payg/etc/motd

```diff
@@ -12,8 +12,8 @@ is configured to match its DNS name.
 To setup SUSE Manager please execute: sudo /sbin/yast2 susemanager_setup
 
 {INCLUDES}
-Management and Config: https://www.suse.com/suse-in-the-cloud-basics
 Documentation: https://www.suse.com/documentation/suse_manager/
-Community: https://community.suse.com/
+PAYG Guide: https://documentation.suse.com/suma/4.3/en/suse-manager/specialized-guides/payg/payg-overview.html
```

Comment:
Same as in BYOS, although 4.3 is the only supported PAYG version at the moment and we don't now whether that link will work for the next SUSE Manager version, so maybe not worth changing at this point.